ui/component/membershipBadge/index.js

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { doOpenModal } from 'redux/actions/app';
+import { selectUserValidMembershipForChannelUri } from 'redux/selectors/memberships';
+
+import MembershipBadge from './view';
+
+const select = (state, props) => {
+  const { uri } = props;
```

Comment:
:small_red_triangle: Membership badges aren't appearing

I confirmed with Anthony that it should at least appear in chat/comment, but I don't see it.  Even tried rebuilding with `pk_live`.

<img width="200" alt="image" src="https://user-images.githubusercontent.com/64950861/190196990-c3c07647-69b8-408d-b32b-dc2d632a710a.png">